sdk/core/System.ClientModel/src/Multipart/MultipartContent.cs

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable disable
+namespace System.ClientModel
+{
+    /// <summary>
+    /// A Content in multipart format.
+    /// </summary>
+    public class MultipartContent : IDisposable
```

Comment:
Please put this in Azure.Core.Experimental for first beta release.

Reply:
@annelo-msft  This is in `System.ClientModel` library which will be common library for both azure and non-azure.

Reply:
Types in the System. namespace can also be added to Azure.Core.Experimental in order to beta them prior to GA.